data/events/scripts/player.lua

```diff
@@ -94,6 +94,32 @@ function Player:onMoveCreature(creature, fromPosition, toPosition)
 end
 
 function Player:onReportRuleViolation(targetName, reportType, reportReason, comment, translation)
+	local name = self:getName()
+	local pendingReport = function () local f = io.open(string.format("data/reports/players/%s-%s-%d.txt", name, targetName, reportType), "r") ; if f then io.close(f) return true else return false end end
```

Comment:
@vankk when i did it, i was thinking in prevent some kind of bot or dos attack to create a massive number of files through reports packets.